ui/src/app/exam/editor/publication/collaborative-exam-owner-picker.component.ts

```diff
@@ -63,18 +64,18 @@ import type { Exam } from '../../exam.model';
                 @for (owner of exam.examOwners; track owner) {
                     {{ owner.email }}
                     <button
-                        class="btn btn-sm btn-light"
+                        class="btn btn-sm btn-link px-0"
```

Comment:
Nappuloiden ei tulis näyttää linkeiltä, jos ne ei vie uuteen näkymään. Näin saavutettavuuden kannalta. (CSCEXAM-1145)

Reply:
Mutta kun nämä ei näytä linkeiltä (koska niissä ei ole mitään tekstiä, ainoastaan ikoni).

Reply:
Ahaa. 
`bg-transparent`
Sopisko tuo?

Reply:
Möh ei sekään kyl ihan loppuun asti toimi. Focus ei näytä mitään.

Reply:
Juu en kyl keksi parempaa, mut mikä se syy sille läpinäkyvyydelle on?